runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesPlugin.java

```diff
@@ -56,19 +57,25 @@
 import net.runelite.client.ui.overlay.OverlayManager;
 
 @PluginDescriptor(
-	name = "Attack Styles",
-	description = "Show your current attack style as an overlay",
-	tags = {"combat", "defence", "magic", "overlay", "ranged", "strength"}
-)
+		name = "Attack Styles",
```

Comment:
Revert this indentation change (https://github.com/runelite/runelite/wiki/Code-Conventions)